scripts/basic-workflow-tests.sh

```diff
@@ -23,13 +23,21 @@ BOLD='\033[1m'
 NC='\033[0m' # No Color
 
 SCRIPTDIR=$(dirname "$0")
+E2E_TEST_DIR="$(pwd)/__e2e_test"
+export XDG_CONFIG_HOME="${E2E_TEST_DIR}/.config"
+export XDG_DATA_HOME="${E2E_TEST_DIR}/.local/share"
+
+# ensure that our mock e2e dirs exist
+for dir in "${XDG_CONFIG_HOME}" "${XDG_DATA_HOME}"; do
+	mkdir -p "${dir}"
+done
 
 step() {
-    echo -e "$BOLD$* - $(date)$NC"
+    echo -e "${BOLD}${*} - $(date)${NC}"
```

Comment:
These changes are not needed, let's not complicate the work and change something that is already working. This is valid for the cosmetic bash changes in all .sh files. Thanks.

Reply:
I reset it locally, will push up in a bit.

Reply:
Resolved

Reply:
It seems like I missed them in this file and only did it in the functional-tests script. Changing this one as well.